.github/workflows/pkgci_test_sharktank.yml

```diff
@@ -62,13 +67,13 @@ jobs:
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
           repository: iree-org/iree-test-suites
-          ref: 56509666d185cdb99aafb4ed4d8532b8f3c54fa3
+          ref: users/eliasj42/change-tolerance-on-llama-test
           path: iree-test-suites
           lfs: true
       - name: Install Sharktank models test suite requirements
         run: |
           source ${VENV_DIR}/bin/activate
-          python -m pip install -r iree-test-suites/sharktank_models/requirements.txt
+          python -m pip install -r iree-test-suites/sharktank_models/requirements-iree.txt
```

Comment:
See https://github.com/iree-org/iree/pull/20302 which also updates the iree-test-suites version used here. We should get your patch(es) landed on `main` there and do a regular update, not pin to a branch.

From that PR, this should also be `python -m pip install iree-test-suites/sharktank_models`. The requirements-iree pulls in _nightly_ IREE packages, when what this test in iree-org/iree should be doing is installing _dev_ IREE packages.

Reply:
See the PR I linked.